modules/framework/shared/src/main/scala/weaver/framework/Fingerprints.scala

```diff
@@ -76,7 +76,7 @@ abstract class WeaverFingerprints[F[_]](implicit F: Sync[F]) {
   /**
    * A fingerprint that searches only for classes extending
    * [[weaver.EffectSuite]]. that have a constructor that takes a single
-   * [[weaver.GlobalResources.Read]] parameter.
+   * `weaver.GlobalResources.Read` parameter.
```

Comment:
scaladoc can't find this symbol:
```
[warn] 81 |  object ResourceSharingSuiteFingerprint extends WeaverFingerprint {
[warn]    |         ^
[warn]    |         No DRI found for query: weaver.GlobalResources.Read
```

Should this be `weaver.GlobalRead`?

Reply:
I think so, indeed

Reply:
As far as I'm concerned, this is the last outstanding item for this PR to be merged 🎉

Reply:
`GlobalRead` came from the cats module, so I went for `weaver.GlobalResourceF.Read` instead.

Reply:
hah, good call, always mix them for one another